### PR TITLE
Code cleanup Pt 1

### PR DESCRIPTION
**Pull request by NoooneyDude**
- Code cleanup
    - Small refactors here and there.
    - Namespacing.
    - Spelling mistakes
    - Follow project guidelines
- Fire-and-forget async tasks to stop compiler warnings.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs [hunk 1]
```diff
@@ -112,7 +112,7 @@ private IEnumerator Release()
 			//TODO wait until the object's velocity is 0 instead of a fixed amount of time!
 			yield return WaitFor.Seconds(0.6f);
 			Spawn.ServerPrefab(facehugger, gameObject.transform.position);
-			Despawn.ServerSingle(gameObject);
+			_ = Despawn.ServerSingle(gameObject);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs:115]: Why is this needed?

**Comment by Bod9001** [comment]
Oh man there is going to be some bad merged conflicts with diynach inventory

File: UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs (hunk 1 above)
Comment (corp-0) [reply UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs:115]: to stop warnings about the call not being awaited

**Comment by corp-0** [comment]
let's merge this last

File: UnityProject/Assets/Scripts/Items/Food/Bread.cs [hunk 2]
```diff
@@ -2,33 +2,31 @@
...
+			if (DMMath.Prob(10))
+			{
+				Spawn.ServerPrefab(mutatedFood, gameObject.AssumedWorldPosServer());
+				_ = Despawn.ServerSingle(gameObject);
+			}
+		}
 
+		//Bread Class has big plans for the future WORK-IN-PROGRESS
+
+		// So say we all.
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Food/Bread.cs:30]: lol

**Comment by NoooneyDude** [comment]
> Oh man there is going to be some bad merged conflicts with diynach inventory

All good, just write over my changes, then you just need to fix namespacing problems with other files - should be easy.

Or the other way around, merge yours first, I guess..

**Comment by AutumnNova** [comment]
> Oh man there is going to be some bad merged conflicts with dynamic inventory

please try and not have a repeat of health
i beg

**Comment by corp-0** [comment]
this pr introduced a compiling error in ElectricalManagerEditor.cs, if you can fix it that'd be appreciated.
